derive/src/to_client.rs

```diff
@@ -134,6 +132,7 @@ fn generate_client_methods(methods: &[MethodRegistration]) -> Result<Vec<syn::Im
 				);
 				client_methods.push(client_method);
 			}
+			_ => continue,
```

Comment:
I put this fallback case here because I am still waiting on an answer for https://github.com/paritytech/jsonrpc/issues/252#issuecomment-512122117 and am currently not handling the `MethodRegistration::Notification` case in the client code generation.

Reply:
Please rather put the variant instead of a wildcard.

Reply:
Sounds good. But what do we do about the client code generation for notification support? Is there anything we need to add to the `MethodRegistration::Notification` case?

Reply:
I have added the explicit variant in place of the wildcard. I am also assuming we do not need to generate any client code for notifications. If we need support for this, please let me know and I can implement it in this PR.